_inc/client/plans/product-selector.jsx

```diff
@@ -22,6 +22,7 @@ import {
 	isFetchingSiteData,
```

Comment:
It's a little unrelated, but we could knock out two birds with one stone here in the `renderTitleSection` method:

1) Relocate the `Which backup option is best for me?` link into the Backups card.
2) Add another plan period toggle, like so:

<img width="1076" alt="Screen Shot 2020-03-30 at 3 36 18 PM" src="https://user-images.githubusercontent.com/4044428/77964400-381b4f80-729c-11ea-83c0-660a3f1512b7.png">

Reply:
YES, please!